data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,90 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
+  {
+    "id": "clothing_technician",
```

Comment:
I want it to replace `technician_shirts` and `technician_pants` groups, but I can't find all places they are used at.